extensions/amp-story/0.1/bookend.js

```diff
@@ -79,6 +85,68 @@ function buildArticle(doc, articleData) {
 
 
 /**
+ * @param {!Document} doc
+ * @param {string} title
+ * @param {string} domainName
+ * @param {string=} opt_imageUrl
+ * @return {!Element}
+ */
+function buildReplayButton(doc, title, domainName, opt_imageUrl) {
+  const root = createElementWithAttributes(doc, 'div', {
+    class: 'i-amphtml-story-bookend-replay',
+  });
+
+  if (opt_imageUrl) {
+    const container = createElementWithAttributes(doc, 'div', {
+      class: 'i-amphtml-story-bookend-replay-image',
+    });
+
+    const iconContainer = doc.createElement('div');
+
+    // Value is constant, so it's OK to use innerHTML here.
+    iconContainer./*OK*/innerHTML = ICONS.refresh;
```

Comment:
Can you do this in CSS instead?  Ultimately, we want to move all our icons there (#11545)

Reply:
Done.